Update miscellaneous SOP #608 (#900)
ebi-ait/hca-ebi-wrangler-central: docs/SOPs/HCA_Metadata_Spreadsheet_Guide_Wranglers.md

* **Experimental design represented as HCA metadata**- How to convert experimental design to a metadata template *(currently wrangler led)*

> * **Making a spreadsheet template**- Generating a spreadsheet using the spreadsheet builder *(wrangler only)* 

Reviewer: ```suggestion
* **Making a spreadsheet template**- Generating a spreadsheet using the spreadsheet builder *(wrangler only)* or using the UI interface (After filling out basic project information) *(wrangler only)*
```

![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_3.png)

### Protocols

Protocols need to be linked to the processes sandwiched between biomaterial and/or file entities.

![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_4.png)

To create a link from one collection protocol to 5 processes between ‘donor organism’ and ‘specimen from organism’, we add the collection protocol ID to the specimen tab. **We always add protocol IDs to the lower entity in the hierarchy**.

![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_5.png)

Although this sample shows the linking of one protocol, multiple protocols can be added to a process.

> ### Bundles

Reviewer: ```suggestion
### Subgraphs
```

Author: I forgot to commit this change as well, but I've seen that the word bundle is still in a number of places.
What is the difference between bundle and subgraph? Do we want to change all occurrences of bundle?

Reviewer: it's not super important but we should be consistent

Basically, there is no difference in the terms but that's how we used to call the subgraphs. There were several discussions and now we call them subgraphs instead of bundles, so our documentation should reflect that

> Datasets are not submitted to the data store, instead datasets are organized into bundles. The splitting is roughly technology dependent as files are grouped together based on requirements for each analysis pipeline run. Bundling for technologies that do not yet have a pipeline should be done consistently with other datasets in the HCA. If you are unsure you should consult with the pipeline developers and other wranglers to define the ideal bundle.

Reviewer 2: We no longer have analysis pipelines, maybe we should remove that part entirely?

Author: I removed the reference with this last commit

> Datasets are not submitted to the data store, instead datasets are organized into bundles. The splitting is roughly technology dependent as files are grouped together based on requirements for each analysis pipeline run. Bundling for technologies that do not yet have a pipeline should be done consistently with other datasets in the HCA. If you are unsure you should consult with the pipeline developers and other wranglers to define the ideal bundle.

Reviewer: ```suggestion
Datasets are not submitted to terra as a whole; instead, datasets are organized into subgraphs. The splitting is roughly technology dependent as files are grouped together based on requirements for each analysis pipeline run. Grouping should be done consistently with other datasets in the HCA. If you are unsure, you should consult with the pipeline developers and other wranglers to define the ideal subgraph.
```

> Ingest creates one bundle for every process linked to a sequence or image file. As mentioned above, one process is automatically created between every entity. Therefore, by default each file will be exported in a separate bundle. Often we do not want this behavior and instead want to bundle differently.

Reviewer: ```suggestion
Ingest creates one subgraph for every process linked to a sequence or image file. As mentioned above, one process is automatically created between every entity. Therefore, by default each file will be exported in a separate subgraph. Often we do not want this behavior and instead want to group differently.
```

> As an example, to create two bundles for a SmartSeq2, paired-end dataset consisting of 2 cells, we need to impose two processes each linked to two files.

Reviewer: ```suggestion
As an example, to create two subgraphs for a SmartSeq2, paired-end dataset consisting of 2 cells, we need to impose two processes each linked to two files.
```
